frontend/src/components/SignInPage/RobotList.tsx

```diff
@@ -0,0 +1,17 @@
+import { components } from "models/schema";
+import React from "react";
+
+/**
+ * Renders information about the user obtained from Microsoft Graph
+ */
+export function RobotList (robots: components["schemas"]["RobotRequest"][]){
+    console.log(robots.length);
+    for (var robot of robots) {
+        console.log(robot.name);
+    }
+    return (
+        <div className="ListOfRobots">
+
+        </div>
+    );
+};
```

Comment:
Will this function be used as is? If not we should remove it and re-add it later